### PR TITLE
Add favicon for dark mode use it on browser dark mode

### PR DESCRIPTION
**Pull request by orestischaral**
Thanks for the help

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -6,10 +6,11 @@
         :key="category.name"
         class="xs:w-full md:w-1/6 px-2 text-center"
       >
-        <NuxtLink :to="`category/${category.name}`" class="">
-          <p
-            class="font-bold text-gray-600 uppercase tracking-wider font-medium text-ss"
-          >
+        <NuxtLink
```
Comment (MrNaif2018) [inline pages/index.vue:9]: What formatter do you use? Just curious

File: pages/index.vue [hunk 2]
```diff
@@ -34,6 +35,30 @@ export default {
...
+    isDarkMode () {
+      if (process.browser) {
+        return window.matchMedia('(prefers-color-scheme: dark)').matches
+      }
+      return false
+    }
+  },
+  head () {
+    return {
+      title: 'BitcartCC Blog',
```
Comment (MrNaif2018) [inline pages/index.vue]: There is no need to duplicate everything from `nuxt.config.js`, only favicon setting is enough.

File: pages/index.vue [hunk 3]
```diff
@@ -34,6 +35,30 @@ export default {
...
+      meta: [
+        { charset: 'utf-8' },
+        { name: 'viewport', content: 'width=device-width, initial-scale=1' },
+        {
+          hid: 'description',
+          name: 'description',
+          content: 'BitcartCC Official Blog'
+        }
+      ],
+      link: [{ rel: 'icon', type: 'image/x-icon', href: this.isDarkMode ? '/favicon_dark.ico' : '/favicon.ico' }]
```
Comment (MrNaif2018) [inline pages/index.vue]: Interesting idea!
But what about using a favicon with white background, which should be visible in dark and light modes perfectly instead?

**Review by MrNaif2018 — CHANGES_REQUESTED** [review]
Thank you very much for your PR! Please take a look at review comments.
Your implementation is very interesting - it might be used for something else too (dark mode detection to enable dark theme for example)
But I think here it's enough to just use a favicon with white background, this one:
https://github.com/bitcartcc/bitcart-media/blob/master/favicon.ico

File: pages/index.vue (hunk 2 above)
Comment (orestischaral) [reply pages/index.vue]: Removed the code since favicon with white background is enough

File: pages/index.vue (hunk 1 above)
Comment (orestischaral) [reply pages/index.vue:9]: Prettier

File: pages/index.vue (hunk 3 above)
Comment (orestischaral) [reply pages/index.vue]: Removed the code since favicon with white background is enough

**Comment by MrNaif2018** [comment]
Looking good now! Could you please also use this favicon (https://raw.githubusercontent.com/bitcartcc/bitcart-media/master/favicon.ico) without additional borders, so that we have consistent favicons in all repos?
изображение

**Review by MrNaif2018 — APPROVED** [review]
Thank you very much for your PR! Merging!

**Comment by MrNaif2018** [comment]
You are now in our contributors list:
https://github.com/bitcartcc/bitcart#contributors-
Thanks again!
As you said you are using prettier. We currently don't have any formatting guidelines set up for our vue/js code, so any suggestions are welcome. We want to set up some CI checks and formatting in our vue repos. I will describe it a bit later, if you want, you can take a look at that

**Comment by orestischaral** [comment]
Sure. I will. 🙂

**Comment by MrNaif2018** [comment]
See bitcartcc/bitcart-admin#13

**Pull request merged**